assessments/projects/cloudevents/self-assessment.md

```diff
@@ -0,0 +1,1051 @@
+  * Off-line reviews are recommended before meetings.
+  * Meetings happen every Thursday at 9 AM PT.
+* Pull Requests will be resolved and merged during official meetings as a result
+  of a motion.
+  * Pull Requests should not be merged if substantial changes have been made in
+    the past 2 days.
+
+##### Pull Request process
+
+Usually, CloudEvents Pull Requests are expected to meet the following
+[criteria](https://github.com/cloudevents/spec/blob/main/docs/SDK-PR-guidelines.md)
+before being merged:
+
+* The author of the PR asks for it to be discussed and reviewed in an upcoming
+  meeting.
+* All comments have been addressed.
+* PRs that have objections/concerns will be discussed off-line by interested
+  parties. A resolution and updated PR will be expected from those talks.
+
+#### Versioning
+
+The release process follows [Semantic Versioning (semver)](https://semver.org/),
+grouping specifications for simultaneous release. Non-breaking changes use
+"patch" updates while breaking changes follow `semver`. Tags are applied to the
+main branch, and additional branches, like core-v2.0, may be created, and then
+deleted post-merger. Creating a release involves updating versions via Pull
+requests, merging, and announcing changes.
+
+### Communications Channels
+
+CloudEvents use different types of [communication
+channels](https://github.com/cloudevents/spec#communications) for different
+purposes.
+
+* Internal communications channels:
+  * [Meetings](https://docs.google.com/document/d/1OVF68rpuPK5shIHILK9JOqlZBbfe91RNzQ7u_P7YCDE/edit?usp=sharing)
+  * Email
+  * GitHub Issues
+* Inbound communications channels:
+  * [CloudEvents CNCF Email Address](cncf-cloudevents@lists.cncf.io)
+  * [CNCF Slack workspace](http://slack.cncf.io/)
+  * [CNCF Meeting Calendar](https://www.cncf.io/community/calendar/)
+* Outgoing communications channels:
+  * [CloudEvents CNCF
+    Subscription](https://lists.cncf.io/g/cncf-cloudevents-sdk)
+
+### Ecosystem
+
+CloudEvents is integrated with different cloud-native technologies and services.
+Its goal is to create interoperability in event-driven architecture.
+
+CloudEvents has many
+[adopters](https://cloudevents.io/#:~:text=CloudEvents%20Adopters), including:
+
+* Adobe I/O Events
+* Azure Event Grid
+* European Commission
+* Google Cloud Eventarc
+* IBM Cloud Code Engine
+* Oracle Cloud
+* VMware Event Broker Appliance
+
+## Security Issue Resolution
+
+### CloudEvents Specification
+
+To report an issue, or to a new idea, contributors can open a new issue on the
+GitHub repository. However, they should check if a similar issue already exists.
+
+CloudEvents also provides a dedicated [email
+address](cncf-cloudevents-security@lists.cncf.io) and a [Security Mailing
+List](https://lists.cncf.io/g/cncf-cloudevents-security/topics) for reporting
+security concerns related to the specification or the SDKs.
+
+### CloudEvents SDKs
+
+#### Responsible Disclosures Process
```

Comment:
Provide links to the Responsible Disclosures Process and Vulnerability Response Process documented in CloudEvents. Additionally, update the content in more details.

Reply:
Thank you for the comment! I added new links to documents CloudEvents has that mention these processes. Unfortunately, they don't have a document outlining these processes in further detail, so I also added a new action item on `Improve the documentation on the Security Response Processes`. I also messaged the CloudEvents team now to ask if they have more details on these processes that I could add to the assessment. I will update it further when they give me more information. Please, let me know if I should make more changes. Thank you!